CONTRIBUTING.rst

```diff
@@ -244,6 +244,11 @@ Here is a simple overview, with pytest-specific bits:
    be released in micro releases whereas features will be released in
    minor releases and incompatible changes in major releases.
 
+   You will need the tags to test locally, so be sure you have the tags from the main repository. If you suspect you don't, set the main repository as upstream and fetch the tags::
```

Comment:
To confirm: is that because of `setuptools-scm`, or something else?

Reply:
Yes, without the tags, the version was showing like 0.1.dev123+xyz. See the pytest Discord where we debugged this: https://discord.com/channels/810210251398840371/810245251502964736/1033019899670437998

Reply:
Thanks!